packages/elements-core/src/components/Docs/HttpService/__tests__/index.spec.tsx

```diff
@@ -59,9 +59,9 @@ describe('HttpService', () => {
 
   it('displays mock server url when embedded in Stoplight Project', async () => {
     render(
-      <Provider host="https://stoplight.io" showMocking project="studio-demo" workspace="elements-examples">
+      <TryItProvider showMocking>
```

Comment:
Yeah, I highly, highly dislike that. :sweat_smile: 

We are using a `TryItProvider` on something that doesn't use `TryIt`, has nothing to do with `TryIt`, etc. :cry:

Reply:
True, initially I assumed it's HttpOperation there. Do you think we should name it differently or not use provider at all?
I assume we would need to get rid of that old Provider anyway and replace it with data we do need. Though it can't be done now so I added that TryItProvider assuming that `showMocking` refers only to TryIt - but it does not